.travis.yml

```diff
@@ -10,8 +10,9 @@ env:
 - STEP=ansible VERSION=2.9.7
 - STEP=ceph-ansible VERSION=v3.2.42
 - STEP=ceph-ansible VERSION=v4.0.22
+- STEP=heroku-cli VERSION=7.42.1-4.2.1 CLI_VERSION=7.42.1 LIB_VERSION=4.2.1
```

Comment:
we can specify only the cli version and hardcode the lib version for now. This will simplify things, and looks like there is no dependency between the two

Reply:
the above wasn't resolved